code/script.js

```diff
@@ -1,51 +1,316 @@
 // Variables that point to selected DOM elements
-const chat = document.getElementById('chat')
+const chat = document.getElementById('chat');
 
-// If you need any global variables that you can use across different functions, declare them here:
 
+// --------------------------------------------------------
+// global variables that you can use across different functions, declare them here:
+// 1a. service offers
+let serviceChoice = "nada";
+let FeeDomain = 0;
+// per year
 
-// Declare your functions after this comment
```

Comment:
wondering if this comment should be under your variable declarations? down at line 28?